libs/net/dal/Services/AVOverviewInstanceService.cs

```diff
@@ -70,10 +69,11 @@ public override AVOverviewInstance Add(AVOverviewInstance entity)
     public override AVOverviewInstance Update(AVOverviewInstance entity)
     {
         var originalSections = this.Context.AVOverviewSections.Where(s => s.InstanceId == entity.Id).ToArray();
-        originalSections.Except(entity.Sections).ForEach(s =>
+        var deletedSections = originalSections.Where(x => !entity.Sections.Any(y => y.Id == x.Id)).ToArray();
```

Comment:
I don't understand the need for the change here?  We use this convention in numerous places.  A `AVOverviewSection` should have an overridden `Equals` function that does this comparison.  I prefer the convention remaining consistent for maintainability.  Take a look at `ReportSection` for an example.

Reply:
The initial change here is for avoiding the unnecessary delete action for some reason even though just for editing data only, however, the change is unnecessary and it is now removed, thanks